### PR TITLE
Increase device run perf test timeout

### PR DESCRIPTION
**Pull request by jmagman**
## Related Issues

https://github.com/flutter/flutter/issues/67753

### CODE REVIEW

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -769,6 +769,10 @@ class PerfTestWithSkSL extends PerfTest {
       _flutterPath,
       <String>[
         'run',
+        if (deviceOperatingSystem == DeviceOperatingSystem.ios)
```
Comment (jonahwilliams) [inline dev/devicelab/lib/tasks/perf_tests.dart:772]: This also needs to apply to the startup test and whatever the SkSL test is

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 2]
```diff
@@ -769,6 +769,10 @@ class PerfTestWithSkSL extends PerfTest {
...
+          ...<String>[
+            '--device-timeout', '10',
```
Comment (liyuqian) [inline dev/devicelab/lib/tasks/perf_tests.dart:774]: Do we also have to apply this to `flutter drive`, or is the timeout there already large enough?

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 1 above)
Comment (jmagman) [reply dev/devicelab/lib/tasks/perf_tests.dart:772]: Done.

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 2 above)
Comment (jmagman) [reply dev/devicelab/lib/tasks/perf_tests.dart:774]: Added a timeout to that as well.

**Comment by jmagman** [comment]
This will definitely increase latency.  Let's see if it reduces flakiness enough to make it worth it.

File: dev/devicelab/lib/framework/utils.dart [hunk 3]
```diff
@@ -432,8 +433,23 @@ Future<String> eval(
 }
 
 List<String> flutterCommandArgs(String command, List<String> options) {
+  // Commands support the --device-timeout flag.
```
Comment (jonahwilliams) [inline dev/devicelab/lib/framework/utils.dart:436]: uber-nit, use a set `{'attach', 'devices', ...}`

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**